<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientTab</class>
 <widget class="QWidget" name="ClientTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="RemoteTextLabel">
        <property name="text">
         <string>Remote Host</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="RemoteClipboard">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CopyClipboardButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Copy to Local Clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RemoteSelectionTitle">
        <property name="text">
         <string>Selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="RemoteSelection">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CopySelectionButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Copy to Local Clipboard</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
